packages/forms/resources/views/components/wizard/step.blade.php

```diff
@@ -12,6 +12,11 @@
 @endphp
 
 <div
+    x-init="
+        if ($el.querySelector('[autofocus]')) return
+
+        $el.setAttribute('tabindex', '0')
+    "
```

Comment:
```suggestion
    x-init="$el.querySelector('[autofocus]') && $el.setAttribute('tabindex', '0')"
```

Reply:
This should be terser and still work right?
Actually, can't we just use `x-bind:tabindex` instead?

Reply:
I'll test both and let you know.